core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -2377,6 +2386,12 @@ components:
               $ref: '#/components/schemas/EcdsaSecp256k1PublicKey'
             stake_vault:
               $ref: '#/components/schemas/EntityReference'
+            unstake_vault:
+              $ref: '#/components/schemas/EntityReference'
+            liquidity_token:
```

Comment:
To match our documentation: https://learn.radixdlt.com/article/how-will-staking-work-on-babylon
This should be `liquid_stake_unit_resource_address`

And perhaps this should be `unstake_claim_token_resource_address`